### ISSUE TITLE
Keyboard navigation for TreeGrid

### ISSUE THREAD

**Issue by pleku** [vaadin/framework#8758]

**Comment by pleku** [comment]
Pending UX

**Comment by pleku** [comment]
The conclusion for UX is, that the cell based keyboard navigation should be removed for TreeGrid and use instead row based navigation, where up and down moves to next / previous row, and left and right arrows either collapse, expand or go to parent. The latter happens when the current node is not expanded, otherwise it is collapsed.

This is the basic standard way ™ it works for every single hierarchical data listing we quickly tested.
